proxy/http/remap/unit-tests/plugin_testing_common.cc

```diff
@@ -46,7 +46,7 @@ getTemporaryDir()
   tmpDir /= "sandbox_XXXXXX";
 
   char dirNameTemplate[tmpDir.string().length() + 1];
-  sprintf(dirNameTemplate, "%s", tmpDir.c_str());
+  snprintf(dirNameTemplate, sizeof(dirNameTemplate), "%s", tmpDir.c_str());
```

Comment:
Likewise, I think this should be `memcpy(dirNameTemplate, tmpDir.c_str(), sizeof(dirNameTemplate));`

Reply:
A lot of nice changes.  Thanks!